utility/src/main/java/org/oppia/util/firebase/CrashLogger.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.util.firebase
+
+import java.lang.Exception
+
+/**
+ * Interface that enables exception logging.
```

Comment:
The 'Interface that' is unnecessary, and slightly distracting (calling out details in a documentation comment emphasizes their importance--is it actually important that this is an interface?). Also, this doesn't describe what the logger is meant to do. Consider an alernative:

```Logger for tracking caught exceptions. Note that this utility may later upload the exceptions to remote services, or log the exception to a file on disk.```